src/components/Projects/TrendingProjectCard.tsx

```diff
@@ -140,88 +148,79 @@ export default function TrendingProjectCard({
             >
               {metadata.name}
             </h2>
+          ) : (
+            <Skeleton paragraph={false} title={{ width: 120 }} />
```

Comment:
**suggestion** we can use the `active` prop to achieve an (arguably) better loading animation: https://ant.design/components/skeleton/#API